### PR TITLE
setup utility files

### PR DESCRIPTION
**Pull request by yoshuawuyts**
Creates a README, preps cargo.toml for release, sets up contributing.md, and CI. Thanks!

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -2,6 +2,17 @@
 name = "remem"
 version = "0.1.0"
 authors = ["Ryan Levick <ryan.levick@gmail.com>"]
```
Comment (rylev) [inline Cargo.toml]: @yoshuawuyts feel free to add yourself to this list.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,66 @@
...
+`crossbeam`.
 
-item.push(1);
-item.push(2);
-item.push(3);
-```
+## License
+
+<sup>
+Licensed under either of <a href="LICENSE-APACHE">Apache License, Version
```
Comment (rylev) [inline README.md:67]: I believe we'll need to add the actual license text to the repo?

**Review by rylev — CHANGES_REQUESTED** [review]
Small changes. Overall looks great! Thanks so much!

File: README.md (hunk 2 above)
Comment (yoshuawuyts) [reply README.md:67]: oh yeah you're right; oops!

File: Cargo.toml (hunk 1 above)
Comment (yoshuawuyts) [reply Cargo.toml]: :tada:

**Comment by yoshuawuyts** [comment]
@rylev implemented feedback; also added examples to the `lib.rs` file and a note on how `clear` works. Hope this is all good now!

**Review by rylev — APPROVED** [review]
❤️

**Pull request merged**